packages/helm-charts/blockscout/templates/blockscout-data-migration.job.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: batch/v1
+kind: CronJob
```

Comment:
Why CronJob instead of Job, if the trigger is always manual

Reply:
afaik Jobs can only be created from CronJobs

Reply:
I took the description from the link you provided literally

<img width="595" alt="image" src="https://user-images.githubusercontent.com/85586/165762566-e644237e-a794-4fe6-9f7a-ba624dee47cd.png">

Reply:
But it's an optional argument :)
<img width="520" alt="Screenshot 2022-04-28 at 15 32 16" src="https://user-images.githubusercontent.com/4222953/165763880-c78ab121-134c-4bd1-85c8-595dba7b02c1.png">

Reply:
It's the only argument I want to use :) 

The entire purpose of this resource is to act as a template to create new jobs from.

Reply:
Okay thanks for clarifying, this is what I missed before: the `kubectl create job` command doesn't accept a file argument except for CronJobs.

I still think there are too many workarounds here. Would it be possible to instead use `kubectl apply -f <file>` (declarative instead of imperative paradigm) with a Job object? Or is anything in the command you used to test this not transferrable?

And if not and CronJob is the simplest setup, can you document that for future reference?

Reply:
Cool thanks for the added context. I would then suggest documenting
- that this should be a Job but is not because of limitations of using celotool -> that can go into a NOTES.txt file (https://helm.sh/docs/chart_template_guide/notes_files/) or into a header comment inside the chart
- make it clear that the schedule that results in "run this CronJob never" is intentional by adding a comment close to it